### PR TITLE
Disable CC for `--export-keys`

### PR DESCRIPTION
**Pull request by 6hundreds**
Fixes https://github.com/gradle/gradle/issues/24461

### CODE REVIEW

File: subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheDependencyResolutionFeaturesIntegrationTest.groovy [hunk 1]
```diff
@@ -608,6 +608,24 @@ class ConfigurationCacheDependencyResolutionFeaturesIntegrationTest extends Abst
         new MavenLocalRepo() | _
     }
 
+    def "disables configuration cache when --export-keys is used"() {
```
Comment (6hundreds) [inline subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheDependencyResolutionFeaturesIntegrationTest.groovy:611]: I'm not quite sure about this test is covering 100% what we want. Ideally it should export some keys, but configuration for exporting is rather complex, and considering this solution is short-term, I'm decided not to adding it. 
Opinions are welcome

Comment (bamboo) [reply subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheDependencyResolutionFeaturesIntegrationTest.groovy:611]: That's good enough.

**Review by bamboo — APPROVED** [review]
👍

**Comment by bamboo** [comment]
@bot-gradle test and merge

**Pull request merged**